boards/arm/xiao_ble/xiao_ble.dts

```diff
@@ -139,30 +139,34 @@
 		compatible = "fixed-partitions";
 		#address-cells = <1>;
 		#size-cells = <1>;
-
-		/* The size of this partition ensures that MCUBoot can be built
-		 * with an RTT console, CDC ACM support, and w/o optimizations.
-		 */
-		boot_partition: partition@0 {
-			label = "mcuboot";
-			reg = <0x00000000 0x00012000>;
-		};
-
-		slot0_partition: partition@12000 {
-			label = "image-0";
-			reg = <0x00012000 0x00069000>;
+		
+		sd_partition: partition@0 {
+			label = "softdevice";
+			reg = <0x00000000 0x00026000>;
```

Comment:
Is it intended to be 0x00026000, while the next partition starts at 0x00027000?

Reply:
Ah, no, that should be 0x00027000. I will fix.